frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -763,8 +763,13 @@ export const experimentLogic = kea<experimentLogicType>([
         },
         updateExperimentSuccess: async ({ experiment }) => {
             actions.updateExperiments(experiment)
-            actions.loadMetricResults()
-            actions.loadSecondaryMetricResults()
+            if (experiment.start_date) {
```

Comment:
i think do the same at `setExperimentStatsVersion` ? Not sure if that can only happen after the experiment starts running though